skpro/regression/multiquantile.py

```diff
@@ -0,0 +1,335 @@
+"""Multiple quantile regression"""
+
+__all__ = ["MultipleQuantileRegressor"]
+__author__ = ["Ram0nB"]
+
+import numpy as np
+import pandas as pd
+from joblib import Parallel, delayed
+from sklearn.base import clone
+
+from skpro.regression.base import BaseProbaRegressor
+
+
+class MultipleQuantileRegressor(BaseProbaRegressor):
+    """Multiple quantile regressor.
+
+    A tabular quantile regressor typically regresses a single probability level,
+    however, often more than one quantile probability is of interest. Therefore, this
+    multiple quantile regressor wraps multiple quantile regressors for multiple quantile
+    probabilities so that each probability has one corresponding regressor. After
+    fitting, all quantile regressors can be used to make probabilistic predictions.
+
+    In `fit`, for every probability in alpha, the quantile_regressor is cloned and the
+    probability is set. Subsequently all regressors are fitted.
+
+    For every desired probability in the probabilistic prediction methods, the fitted
+    regressor that is closest to the desired probability is selected to make the
+    corresponding prediction.
+
+    Parameters
+    ----------
+    mean_regressor : Sklearn compatible regressor or None, default=None
+        Tabular mean regressor for `predict`. Can't predict mean if not provided.
+    quantile_regressor : Sklearn compatible quantile regressor or None, default=None
+        Tabular quantile regressor for `predict_quantiles`, `predict_interval` and
+        `predict_proba`. Can't do probabilistic prediction if not provided.
+    alpha_name : str or None, default=None
+        Alpha parameter name that sets the quantile probability level for the
+        quantile_regressor. Can't do probabilistic prediction if not provided.
+    alpha : list with float or None, default=None
```

Comment:
what happens if `alpha` is None? It looks to me like it will just fail then. So, should `alpha` not be compulsory?